src/ui/FilterForm.jsx

```diff
@@ -64,70 +64,78 @@ function FilterForm({
 	// Otherwise the user cannot submit the filter form (using the "Set filters" button).
 
 	// These change handlers implement custom logic to create a dependency between outer and inner
-	// checkboxes. Feel free to ignore, its a crude / partially hardcoded implementation anyway.
+	// checkboxes. Feel free to ignore, its a crude / partially hard-coded implementation anyway.
```

Comment:
If you want to spellcheck the whole thing, change its to it's then :)

Reply:
Done